### PR TITLE
WIP: Proposal: Skip all AppVeyor commits via message

### PR DESCRIPTION
**Pull request by jakirkham**
**DO NOT MERGE!**

Attempts to match all commit messages and makes that the method for skipping AppVeyor builds.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@
 # file, make changes to conda-forge.yaml and/or recipe/meta.yaml, and run
 # "conda-smithy regenerate".
 
+# Skipping commits with particular message or from user
+skip_commits:
+  message: .*
+  author: .*
```
Comment (ocefpaf) [inline appveyor.yml]: Oh. Now I get it!

Comment (jakirkham) [inline appveyor.yml]: Yeah, it's not super elegant. Though if we can't get the ideal case, this is still an improvement.

**Comment by ocefpaf** [comment]
I wonder how difficult would it be to write a git hook that adds `[skip appveyor]` to the commit message when `Skip: True  # [win]` is present.

That is not a general solution as we would need to send this hook to every maintainer...

**Comment by ocefpaf** [comment]
@jakirkham I guess we can close this, right?

**Comment by jakirkham** [comment]
Yep, closed. Thanks for the reminder.

**Pull request closed without merging**